schema/v1.1.0/dataset_config_models_extended.py

```diff
@@ -0,0 +1,321 @@
+from __future__ import annotations
+
+import asyncio
+import re
+from typing import Dict, List, Optional, Set, Tuple
+
+import aiohttp
+from async_lru import alru_cache
+from dataset_config_models import (
+    Annotation,
+    AnnotationConfidence,
+    AnnotationEntity,
+    AnnotationObject,
+    AnnotationSource,
+    Author,
+    Container,
+)
+from pydantic import field_validator, model_validator
+from typing_extensions import Self
+
+CELLULAR_COMPONENT_GO_ID = "GO:0005575"
+
+running_network_validation = False
+
+# ==============================================================================
+# Annotation Object Validation
+# ==============================================================================
+
+
+@alru_cache
+async def validate_go_id(go_id: str) -> Tuple[bool, dict]:
+    url = f"https://api.geneontology.org/api/ontology/term/{go_id}"
+    headers = {"Accept": "application/json"}
+    async with aiohttp.ClientSession() as session, session.get(url, headers=headers) as response:
```

Comment:
creating a aiohttp session has negligible overhead, so it's fine to create one everytime